style.css

```
@@ -1,44 +1,112 @@
-body{
-    background-color: grey;
-    display: flex;
-    justify-content: center;
-    flex-direction: column;
-    align-items: center;     
-}
-.controler{
-    display: flex;
-    align-items: center;    
-    flex-direction: column;
-    justify-content: center;          
-    width: 40%;
-    color: ivory;    
-}
-.units{
-    display: flex;   
-    flex-direction: column;
-    align-content: center;
-    border-width: 2px;
-    border:darkblue;
-}
-#weatherDisplay{
-    display: flex;
-    flex-direction: row;
-    width: 90%;
-    justify-content: space-evenly;
-    border: 3px;
+/* *{
+  border: 1px solid red !important;
+} */
+body {
+  background-color: rgb(176, 183, 190);
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  font-family: "Roboto Condensed", sans-serif;
+}

```

Deja un enter entre cada clase en el estilo.
```suggestion
}

```